packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -205,6 +206,14 @@ export function TransferPanel() {
     gasSummary
   })
 
+  const { color: sourceChainUIcolor } = getBridgeUiConfigForChain(
```

Comment:
```suggestion
  const { color: sourceChainUiColor } = getBridgeUiConfigForChain(
```
nit, same for destination

Reply:
i briefly thought about this
but `Ui` makes me cringe T_T ok i'll change